sentry-core/src/session.rs

```diff
@@ -257,9 +255,13 @@ impl SessionFlusher {
             attributes: session_update.attributes.clone(),
         });
 
-        let started = session_update
+        let duration = session_update
             .started
-            .duration_trunc(ChronoDuration::minutes(1))
+            .duration_since(SystemTime::UNIX_EPOCH)
+            .unwrap();
+        let duration = (duration.as_secs() / 60) * 60;
```

Comment:
What is this intended to do?

Reply:
Integer division is truncating, so this should round down to the nearest minute.

Reply:
Yeah I figured, I just don't really understand why we're rounding it when we output seconds anyway.

Reply:
When we pre-aggregate sessions on the client, we group them into minute buckets based on the truncated systemtime ;-)

Reply:
I see, thanks for the explanation :)